### DISCUSSION TITLE
Hotelling's T2 equivalence in functional data?

### DISCUSSION THREAD

**Discussion by hovinh** [GAA-UAM/scikit-fda#400] [Q&A]
Hi scikit-fda team,

I would like to extend existing MVA methods from conventional tabular to time series data type, leveraging upon FDA method. 
In particular, I'm interested in monitoring chart. Commonly I employ Hotelling T's chart to gather multivariate data to a scalar metric, namely Hotelling T2 that can be computed via PCA; I set a control limit on this 1-dimensional chart for anomaly detection. If an outlier detected, I can even diagnose with "inverse transform" on PCA to pin point the feature that primarily drive this anomaly. 

Formally, given a dataset X, each sample X_i = (X_i1, X_i2,... X_ij) with j is the number of features. We can compute T_i = f(X_i, X) which is an "artificial" scalar metric to summarize the behaviour of all features of a sample.

I wonder if FDA literature has something similar for time-series data? If so, could you please provide me with the keyword and known package that implements them already?

Thank you.

Sincerely,
Vinh

**Comment by vnmabus** [comment]
Is this what you wanted?

https://fda.readthedocs.io/en/latest/modules/inference/autosummary/skfda.inference.hotelling.hotelling_t2.html#skfda.inference.hotelling.hotelling_t2

**Reply by hovinh** [reply]
Thank you vnmabus. To clarify further, let's say if I want to utilize Hotelling's T2 in form of **monitoring chart**, I can create 2 functional data, `X` which is the training dataset, and `x` which is a new trace. The computed T2 value for `x` is as follows? 
```python
t2 = hotelling_t2(X, x)
```

**Reply by vnmabus** [reply]
Sorry, I have never used monitoring charts, even in the multivariate case. You will need to check that the functional Hotelling T2 is the analog of the thing that you compute in the multivariate case (the reference for the functional Hotelling is in the linked documentation).

**Reply by hovinh** [reply]
Thank you :D

**Reply by vnmabus** [reply]
BTW, maybe you are interested in page 24 of https://iwfos2021.sci.muni.cz/media/3326670/iwfos2021-book-of-abstracts.pdf. I attended the conference, but I must confess that I didn't pay much attention to this one 😅.

**Reply by hovinh** [reply]
Thank you. I will have a look at it :D
